src/xrpld/app/tx/detail/OfferStream.cpp

```diff
@@ -272,6 +272,19 @@ TOfferStreamBase<TIn, TOut>::step()
             continue;
         }
 
+        bool const deepFrozen = isDeepFrozen(
```

Comment:
should be gated by the deepfreeze amendment. also should test pre/post amendment in unit test if not already